From 2e6430ad285f8cdceddde86a80651905a125e2c4 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Fri, 14 Aug 2015 14:56:32 +0800
Subject: [PATCH 0211/5242] MLK-11363-2 ARM: dts: imx7d-sdb: enable epxp in
 device node

commit  adfd0f22abe83b3bc6f90d62bb37351c1224148b from
https://source.codeaurora.org/external/imx/linux-imx.git

Enable epxp in device node by default on imx7d-sdb board.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index 41d0c30..037911a 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -183,6 +183,10 @@
 	};
 };
 
+&epxp {
+	status = "okay";
+};
+
 &fec1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_enet1>;
-- 
1.7.9.5

